modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/status_effects/aroused.dm

```diff
@@ -5,35 +5,37 @@
 	alert_type = null
 
 /datum/status_effect/aroused/tick()
+	if(owner.stat >= DEAD)
+		return
+
 	var/mob/living/carbon/human/affected_mob = owner
 	var/temp_arousal = -0.1
 	var/temp_pleasure = -0.5
 	var/temp_pain = -0.5
-	if(affected_mob.stat != DEAD)
 
-		var/obj/item/organ/external/genital/testicles/balls = affected_mob.getorganslot(ORGAN_SLOT_TESTICLES)
-		if(balls)
-			if(balls.internal_fluids.holder_full())
-				temp_arousal += 0.08
+	var/obj/item/organ/external/genital/testicles/balls = affected_mob.getorganslot(ORGAN_SLOT_TESTICLES)
+	if(balls)
+		if(balls.internal_fluid_full())
+			temp_arousal += 0.08
 
-		if(HAS_TRAIT(affected_mob, TRAIT_MASOCHISM))
-			temp_pain -= 0.5
-		if(HAS_TRAIT(affected_mob, TRAIT_NEVERBONER))
-			temp_pleasure -= 50
-			temp_arousal -= 50
+	if(HAS_TRAIT(affected_mob, TRAIT_MASOCHISM))
+		temp_pain -= 0.5
+	if(HAS_TRAIT(affected_mob, TRAIT_NEVERBONER))
+		temp_pleasure -= 50
+		temp_arousal -= 50
```

Comment:
All of these numbers should realistically be defines. Because here right now they mean basically nothing.